grails-app/services/streama/TheMovieDbService.groovy

```diff
@@ -159,8 +161,7 @@ class TheMovieDbService {
       apiCacheData["$type:$name"] = data
     }
     catch(e) {
-      HttpURLConnection conn = url.openConnection()
-      throw new Exception("TMDB request failed with statusCode: " + conn?.responseCode + ", responseMessage: " + conn?.responseMessage + ", url: " + requestUrl)
```

Comment:
why remove this exception text? its pretty useful for the content manager to know whats going on

Reply:
I think:
1 . the `e` contains statuscode;
2. calling url.openConnection() twice not a good solution.

Reply:
all right lets try :)